docs/pages/desktop-access/active-directory.mdx

```diff
@@ -1,158 +1,173 @@
 ---
-title: Desktop Access with Active Directory
-description: Integrate the Teleport Desktop Service with your Active Directory domain
+title: Configure access for Active Directory with scripts
+description: Demonstrates how to integrate the Teleport Windows Desktop Service with your Active Directory domain using predefined scripts.
 videoBanner: v36tlB8D5kk
 ---
 
-This guide will help you configure Teleport and Active Directory in order to
-provide secure, passwordless access to Windows desktops.
+This guide demonstrates how to use predefined scripts to configure Teleport and Active Directory 
+to provide secure, passwordless access to Windows computers.
```

Comment:
"predefined scripts" sounds like it could be the user who defines them. Is there another term we could use to capture the fact that the integration process is automated?